src/driver_micarray_alsa.cpp

```diff
@@ -29,14 +29,11 @@ namespace matrix_malos {
 
 bool MicArrayAlsaDriver::ProcessConfig(const pb::driver::DriverConfig& config) {
   pb::io::MicArrayParams micarray_config(config.micarray());
-
   mics_.SetGain(static_cast<int16_t>(micarray_config.gain()));
```

Comment:
What about setting some defaults here, in case no values are set in the config proto ?

Reply:
I manage it here: https://github.com/matrix-io/matrix-creator-hal/blob/master/cpp/driver/microphone_array.cpp#L100 and keep the initial values